website/content/en/docs/getting-started/getting-started-with-karpenter/prometheus-values.yaml

```diff
@@ -15,6 +15,10 @@ extraScrapeConfigs: |
           names:
           - kube-system
       relabel_configs:
-      - source_labels: [__meta_kubernetes_endpoint_port_name]
-        regex: http-metrics
+      - source_labels:
+          [
```

Comment:
Good catch! nit: Does it make sense to make this a standard yaml array to make the formatting consistent.

```yaml
- source_labels:
    - __meta_kubernetes_endpoints_name
    - __meta_kubernetes_endpoint_port_name

```